draft-ietf-tls-external-psk-importer.md

```diff
@@ -195,7 +203,33 @@ PSKs for TLS 1.2. Indeed, this is necessary for incremental deployment.
 
 # Security Considerations
 
-DISCLAIMER: This is a WIP draft and has not yet seen significant security analysis.
+The Key Importer security goals can be roughly stated as follows: avoid PSK re-use across
+KDFs while properly authenticating endpoints. When modeled as computational extractors, KDFs
+assume that input keying material (IKM) is sampled from some "source" probability distribution
+and that any two IKM values are chosen independently of each other {{Kraw10}}. This
+source-independence requirement implies that the same IKM value cannot be used for two different
+KDFs.
+
+PSK-based authentication is functionally equivalent to session resumption in that a connection
+uses existing key material to authenticate both endpoints. Following the work of
+{{?BAA15=DOI.10.14722/ndss.2015.23277}}, this is a form of compound authentication. Loosely
+speaking, compound authentication is the property that an execution of multiple authentication
+protocols, wherein at least one is uncompromised, jointly authenticates all protocols.
```

Comment:
"Specifically the uncompromised protocol execution proves that all the other protocols bound to it are authentic."
Should we mention that some authentication protocols only authenticate earlier protocols, and some only authenticate later ones? This fact is derivable from the reference, but is not explicitly mentioned.

Reply:
Hmm, is that detail relevant here? (I don't this so, but you are the expert!)